pkg/config/config.go

```diff
@@ -161,6 +163,9 @@ type Config struct {
 	ImagePullPolicy   string            `json:"ImagePullPolicy" mapstructure:"ImagePullPolicy"`
 	ImagePullSecrets  string            `json:"ImagePullSecrets" mapstructure:"ImagePullSecrets"`
 	CustomAnnotations map[string]string `json:"CustomAnnotations,omitempty" mapstructure:"CustomAnnotations"`
+
+	// ProgressUpdatesPort is the port on which the Sonobuoy worker will listen for status updates from its plugin.
+	ProgressUpdatesPort string `json:"ProgressUpdatesPort,omitempty" mapstructure:"ProgressUpdatesPort"`
```

Comment:
The line has been updated here; not sure why it wasn't in the line you commented on. I'm unable to reply to your comment as well since I think it is on an old commit that isn't referenced now.